codegen/golang.go

```diff
@@ -19,9 +20,11 @@ type TableOptions func(*TableDefinition)
 //go:embed templates/*.go.tpl
 var TemplatesFS embed.FS
 
-func valueToSchemaType(v reflect.Type) (schema.ValueType, error) {
+func defaultTypeTransformer(v reflect.Type) (schema.ValueType, error) {
```

Comment:
Is the default transformer exposed to consumers? Otherwise any consumer that overrides will need to re-implement it

Reply:
Good catch. I'll bring the override behaviour back.

Reply:
done

Reply:
Actually second thought I removed the implicit override behaviour  and made `DefaultTypeTransformer` public so the user can decide for example on the order and so on. this also make the sdk code cleaner and more explicit.